xwiki-platform-distribution/xwiki-platform-distribution-war-dependencies/pom.xml

```diff
@@ -662,6 +662,18 @@
       <version>${project.version}</version>
     </dependency>
 
+    <!-- Must be installed at the farm level because xwiki-platform-mentions-default must be installed at farm level. 
```

Comment:
Putting this extension here does not "install it at farm level", it makes it a core extension.

Reply:
Comment updated.